### ISSUE TITLE
How to add additional input field on schema?

### ISSUE THREAD

**Issue by marcobustillo** [graphql-compose/graphql-compose#200]
How do i add additional field on input type schema? Is there any functions similar to addFields?

**Comment by nodkz** [comment]
Yep Input type composer has the `addFields` method: https://graphql-compose.github.io/docs/api/InputTypeComposer.html#addfields
